doc/index.rst

```diff
@@ -0,0 +1,22 @@
+.. _index:
+
+Welcome to npmx zephyr PMIC support repository
+##############################################
+
+Nordic Semiconductor announces multi-function `Power Management`_ IC (PMIC) with unique system management features.
+
+:ref:`npmx-zephyr <index>` is a repository with zephyr-like samples for Nordic Semiconductor `npmx library`_.
+
+Zephyr also provides a variety of application samples and demos.
+Documentation for those is available in Zephyr's `Samples and Demos`_ section.
+For very simple samples, see the `Basic Samples`_.
+Those samples are a good starting point for understanding how to put together your own application.
```

Comment:
No need to explain what other zephyr examples. Just linking to Zephyr's getting started would be ok.